### ISSUE TITLE
integrating shapetrees in solid

### ISSUE THREAD

**Issue by elf-pavlik** [solid/data-interoperability-panel#58]
Shape Trees Specification defines  (discovery of container Shape Tree Metadata)[https://shapetrees.github.io/specification/spec#discover] using specific link relation. Solid server would be responsible for naming this resource and managing it as one of auxiliary resources. It seems required that auxiliary resources section of solid spec should specifically address requirements of Container Shape Tree Metadata.

**Comment by justinwb** [comment]
@elf-pavlik yes you have it right. the submission for server-side shape tree support will include a shapetree auxiliary resource in addition to the expected server-side logic

**Comment by elf-pavlik** [comment]
I'm not sure which parts for shape tree you can make optional, but auxiliary resource seems a MUST for clients to be able to use shape trees.

**Comment by justinwb** [comment]
@elf-pavlik Yes that's right. Technically you could use the RDF description (rel=describedby) as a fallback, but that's not really appropriate. I should've said in the prior comment "the submissions for **solid** server-side shape tree support". We plan to write that up and submit it to solid/specification.
